cmd/format-erasure.go

```diff
@@ -646,7 +646,7 @@ func saveFormatErasureAll(ctx context.Context, storageDisks []StorageAPI, format
 		}, index)
 	}
 
-	writeQuorum := (len(storageDisks) + 1/2)
+	writeQuorum := ((len(storageDisks) + 1) / 2)
```

Comment:
In-fact the code is correct @jiuker we do not support partial `format.json` writes anymore and this code is executed when its a fresh setup. 

So this code is correct and it should if all drives didn't succeed not 50% of them.  So please change the code to be 

```
reduceWriteQuorumErrs(ctx, g.Wait(), nil, len(storageDisks))
```